### PR TITLE
Configured destination chain as cere

### PR DESCRIPTION
**Pull request by karthikeyan-cere**
Cere-Bridge
Cere-Bridge-1

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,3 +5,4 @@
 - Added home path to logo and title
 - Added polygon configuration to chainbridgeConfig
 - Updated title
+- Configured destination chain as CERE
```
Comment (AndreiNavoichyk) [inline CHANGELOG.md]: Rename to `Set Cere Network as Destination Network`.

**Review by AndreiNavoichyk — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (karthikeyan-cere) [reply CHANGELOG.md]: Updated.

File: src/Components/Pages/TransferPage.tsx [hunk 2]
```diff
@@ -377,15 +377,18 @@ const TransferPage = () => {
...
             <SelectInput
               label="Destination Network"
               className={classes.generalInput}
               disabled={!homeConfig}
-              options={destinationChains.map((dc) => ({
-                label: dc.name,
-                value: dc.chainId,
-              }))}
+              options={destinationChains
+                .filter((dc) => dc.chainId === 1)
```
Comment (AndreiNavoichyk) [inline src/Components/Pages/TransferPage.tsx]: We can not be sure that chainId will be the same on all envs. We should think about more flexible solution.

Comment (karthikeyan-cere) [reply src/Components/Pages/TransferPage.tsx]: No, In all three env the chainId for cere network is set to 1. 
Even in future if we need to differentiate the network (Testnet, Devnet, Mainnet), we can add one more param as NetworkId like Ethereum.

**Review by AndreiNavoichyk — APPROVED** [review]

**Review by AndreiNavoichyk — APPROVED** [review]

**Pull request merged**